src/pint/fitter.py

```diff
@@ -415,24 +415,30 @@ def get_summary(self, nodmx=False):
                         pn, prefitpar.str_quantity(prefitpar.value), "", par.units
                     )
                 elif par.frozen:
-                    if (
-                        par.name == "START"
-                        and prefitpar.value is None
-                        or par.name != "START"
-                        and par.name == "FINISH"
+                    if par.name in ["START", "FINISH"] and prefitpar.value is None:
+                        s += ("{:" + spacingName + "s} {:20s} {:28g} {} \n").format(
+                            pn, " ", par.value, par.units
+                        )
+                    elif par.name in ["START", "FINISH"]:
+                        s += ("{:" + spacingName + "s} {:20g} {:28g} {} \n").format(
+                            pn, prefitpar.value, par.value, par.units
+                        )
```

Comment:
Can these be converted into fstrings? 

Curly brackets can be escaped in fstrings like this:
```
foo = 1.5
f"{foo}  {{bar}}" == "1.5  {bar}"
```

Reply:
possibly.  I didn't write these and didn't want to mess with the formatting.

Reply:
OK..